app/uk/gov/hmrc/soletraderidentificationfrontend/models/PageConfig.scala

```diff
@@ -24,25 +24,21 @@ case class PageConfig(optServiceName: Option[String],
                       enableSautrCheck: Boolean,
                       accessibilityUrl: String,
                       optFullNamePageLabel: Option[String],
-                      labels: Option[JourneyLabels] = None)
+                      optLabels: Option[JourneyLabels] = None)
 
 object PageConfig {
 
-  def apply(optServiceName: Option[String],
-            deskProServiceId: String,
+  def apply(deskProServiceId: String,
             signOutUrl: String,
             enableSautrCheck: Boolean,
             accessibilityUrl: String,
             optFullNamePageLabel: Option[String],
-            welshFullNamePageLabel: Option[String],
-            welshServiceName: Option[String]): PageConfig = {
+            labels: JourneyLabels
```

Comment:
I think with this change the possibility of supplying the welshFullNamePageLabel is gone, which is not as per the requirements. This should stay, Json labels block part for welsh should stay as it is. New english block should have optFullNamePageLabel and english serviceName.
For example the JourneyLabels class could hold all the labels: english and welsh service names, as well as english and welsh full name page labels.